frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -266,12 +277,22 @@ export const PrometheusMultilineUtilizationItem = withDashboardResources<
     namespace,
     isDisabled = false,
   }) => {
+    const { t } = useTranslation();
```

Comment:
wondering if we shouldn't pull trig block (and the same one above) into a standalone function that would just return `UTILIZATION_QUERY_HOUR_MAP`, so we dont repeat ourselves